lib/timeline.js

```diff
@@ -0,0 +1,69 @@
+const guid = require('./utils/guid')
+
+class Timeline {
+  constructor(client) {
+    this.client = client
+  }
+
+  createEventType(applicationId, userId, data, cb) {
+    data['applicationId'] = data['applicationId'] || applicationId
+    return this.client._request(
+      {
+        method: 'POST',
+        path: `/integrations/v1/${applicationId}/timeline/event-types?userId=${userId}`,
+        body: data,
+      },
+      cb,
```

Comment:
This is very minor but I rather like the format you've used in the `updateEventType` where the last line reads `return this.client._request(parameters, cb)`. Thoughts on applying the same format to this function?